Automattic/wp-calypso
# G Suite: Retire G Suite upsell

This pull request removes any code from the G Suite upsell that would be shown in the domain purchase flow prior to https://github.com/Automattic/wp-calypso/pull/55592:

<img width="623" alt="screenshot" src="https://user-images.githubusercontent.com/594356/133410383-f9c3e7f7-044c-453e-b581-05edc1821b98.png">

#### Testing instructions

1. Run `git checkout remove/gsuite-upgrade` and start your server, or open a [live branch](https://github.com/Automattic/wp-calypso/pull/56268#issuecomment-919854105)
2. Open the [Domain Search page](http://calypso.localhost:3000/domains/add)
3. Select any domain in order to add it to your shopping cart
4. Assert that you are still presented with the new upsell (i.e. tge comparison page)
5. Assert that you can still access the former upsell in production at:
```
https://wordpress.com/domains/add/:domain/google-workspace/:site
```
6. Assert that you can no longer access the former upsell locally at:
```
http://calypso.localhost:3000/domains/add/:domain/google-workspace/:site
```
